AutomatedTesting/Gem/PythonTests/scripting/scripting_utils/script_canvas_component.py

```diff
@@ -0,0 +1,133 @@
+        """
+        Function for constructing the SCComponent object. This will make a new entity, add a script canvas component and
+        then open a script canvas file from disk onto the component's source handle value.
+
+        param entity_name: The name you want the entity to have
+        param sc_file_path: The path on disk to the script canvas file
+        param position: optional parameter for the position of the entity
+
+        Returns None
+        """
+        sourcehandle = scriptcanvas.SourceHandleFromPath(sc_file_path)
+
+        self.editor_entity = EditorEntity.create_editor_entity_at(position, entity_name)
+        self.editor_entity.add_components([SCRIPT_CANVAS_UI])
+
+        self.sc_component = self.editor_entity.get_components_of_type([SCRIPT_CANVAS_UI])[0]
+        self.sc_component.set_component_property_value(SCRIPT_CANVAS_COMPONENT_PROPERTY_PATH, sourcehandle)
+
+    def setup_SC_component_existing_entity(self, editor_entity: EditorEntity, sc_file_path: str,
+                                           sc_component_index=0) -> None:
+        """
+        Function for constructing the SCComponent object. This uses an existing entity and loads a new script canvas
+        file into the source handle value
+
+        param editor_entity: The entity we want to configure
+        param sc_file_path: location on disk to the script canvas file
+
+        returns None
+        """
+        sourcehandle = scriptcanvas.SourceHandleFromPath(sc_file_path)
+
+        self.editor_entity = editor_entity
+        self.sc_component = self.editor_entity.get_components_of_type([SCRIPT_CANVAS_UI])[sc_component_index]
+
+        self.sc_component.set_component_property_value(SCRIPT_CANVAS_COMPONENT_PROPERTY_PATH, sourcehandle)
+
+    def set_variable_value(self, variable_name: str, variable_state: VariableState, variable_value) -> None:
+        """
+        Function for setting a value to a variable on the script canvas component.
+
+        Note: The variable must be exposed to the component. this can be done through the script canvas node inspector.
+        variables exposed to the component will be visible by name rather than type.
+
+        param variable_name: the name of the variable
+        param variable_state: used/unused variable
+        param variable_value: the value to assign to the variable
+
+        returns None
+        """
+        component_property_path = self.__make_variable_component_property_path(variable_name, variable_state)
+        print(variable_value)
+        self.sc_component.set_component_property_value(component_property_path, variable_value)
+
+        #validate the change
+        set_value = self.sc_component.get_component_property_value(component_property_path)
+        assert set_value == variable_value, f"Component variable {variable_name} was not set properly"
+
+    def __make_variable_component_property_path(self, variable_name: str, variable_state: VariableState) -> str:
```

Comment:
let's use construct over "make" here.

EG: `_construct_variable_property_path(...)`

Reply:
changed this per feedback